terraform-e2e/main.tf

```diff
@@ -28,6 +28,7 @@ module "en" {
 
   project = var.project
   database_name = "en-verification-${random_string.suffix.result}"
+  redis_name = "verification-cache-${random_string.suffix.result}"
```

Comment:
Please run `terraform fmt` or install an editor plugin that does auto-formatting.

Reply:
done